scripts/demo-content.py

```diff
@@ -1,14 +1,18 @@
 from models import database, Item, User, Address, Order, OrderItem
 from faker import Factory
 from random import seed, randint
+import argparse
 
 
 def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('seed', type=int)
+    parser.add_argument('fakeseed', type=int)
+    args = parser.parse_args()
     database.connect()
-    seed(2)
-
+    seed(args.seed)
     fake = Factory.create('it_IT')
-    fake.seed(99)
+    fake.seed(args.fakeseed)
```

Comment:
Rendiamo questo seed opzionale, e il default sia il classico os.urandom che pero', in questo caso, lo scriviamo in output all'utente.
E quindi l'argomento seed dato con add_argument sia compatibile con un urandom.